docs/production/mobile-push-notifications.md

```diff
@@ -158,7 +158,7 @@ and help us allocate resources towards supporting self-hosted
 installations.
 
 Our use of these statistics is governed by the same ToS and
-Privacy Policy that covers the Mobile Push Notifications Service
+privacy policy that covers the Mobile Push Notifications Service
```

Comment:
I think this should be capitalized as it's referring to a specific thing?  That's generally legal document convention.   And ToS on the previous line should get expanded.